frontend/lib/moratorium-banner.tsx

```diff
@@ -1,43 +1,29 @@
-import React, { Component } from 'react'
+import React, { useState } from 'react'
+import { SimpleProgressiveEnhancement } from './progressive-enhancement';
 
+const MoratoriumBanner = () => {
+    
+    const [isVisible, setVisibility] = useState(true);
 
-type Props = {}
-type State = {
-    isHidden: boolean
-  }
-  
-
-class MoratoriumBanner extends Component<Props,State> {
-    constructor(Props: Props) {
-      super(Props);
-  
-      this.state = {
-        isHidden: false
-      }
-  
-    }
-  
-    closeBanner = () => this.setState({isHidden: true});
-  
-    render() {
-      return (
-        <section className={"jf-moratorium-banner hero is-warning is-small " + (this.state.isHidden && "is-hidden")}>
-          <div className="hero-body">
-            <div className="container">
-                <div className="jf-close-button is-size-5 is-pulled-right" onClick = {this.closeBanner}>✕</div>
-                <p>
-                    <span className="has-text-weight-bold">COVID-19 Update: </span>
-                    JustFix.nyc is still in operation, and we are adapting our products to match new rules put in place during the Covid-19 public health crisis. 
-                    Thanks to organizing from tenant leaders, renters now have stronger protections during this time, including a full halt on eviction cases. 
-                    {' '}<a href="https://www.righttocounselnyc.org/moratorium_faq" rel="noopener noreferrer">
-                        <span className="has-text-weight-bold">Learn more</span>
-                    </a>
-                </p>
-            </div>
-          </div>
-        </section>
-      );
-    }
-  }
+    return (
+    <section className={"jf-moratorium-banner hero is-warning is-small " + (!isVisible && "is-hidden")}>
```

Comment:
Er, I think that if `isVisible` is true, this section's `className` will be `"jf-moratorium-banner hero is-warning is-small false"`, which is probably unintentional.  I don't think this will actually have a detrimental effect since we probably don't have a CSS class called "false", but it's still a bit funky. 

Whenever I have a conditional class in addition to other classes, I use the [`classnames`](https://www.npmjs.com/package/classnames) package (which is already in our `package.json`) since it avoids these kinds of issues and usually makes the code a bit easier to read, but it's up to you.

Reply:
Oh right! I always forget about that package— so useful!